_drafts/schematic_review_checklist_for_firmware_engineer.md

```diff
@@ -0,0 +1,132 @@
+
+<!-- excerpt end -->
+
+This post written in a voice that makes two general assumptions:
+
+ 1. The reader is a firmware engineer.
+ 1. The reader is not doing the schematic layout 
+
+While those two assumptions helped me write this post, I think the learnings/stories in it are still applicable for a variety of setups. e.g.: If the reader is an electrical engineer that wants to understand some firmware concerns. Or if the reader is responsible for both the electrical and firmware design of a project. With either of these assumptions broken, feel free to put on your firmware engineer hat, and read! Hopefully this will be helpful for you.
+
+{% include newsletter.html %}
+
+{% include toc.html %}
+
+## Schematic Review - What's Your Role as a Firmware Engineer
+
+Most of the schematic reviews I've attended are led by electrical engineers, and are focused on the electrical components of the system. There are usually some questions about whether the proper diode was used, or if the capacitors were selected properly. Things that an electrical engineer should care about, but typically fly over my head. As a firmware engineer, I've found it useful to be thinking about what hardware features the code I'll end up writing will need. Starting from that perspective, I can leverage the schematic review as an opportunity to catch design issues while the fix is relatively cheap, before hardware has gone through various rounds of testing, which can be the difference between hitting a schedule or not.
+
+"Schematic review" here is mostly used as an easy blog post title, and a clear time for the firmware and electrical teams to interact. However, the important piece of this post is that you as a firmware engineer work with your electrical engineering counterpart to ensure you have the hardware you need to develop, debug, and deploy your code. And you identify those needs early on in the development cycle with the electrical team, when changes to PCBs are relatively cheap.
+
+Some pre-schematic review checklist items:
+
+ - [ ] Get invited to the schematic review. Really, this is a callout to be working with your electrical engineer. If you've been working with your electrical engineer, providing guidance on what hardware you'll need, being invited to the schematic review should be a given.
+ - [ ] Build your checklist up front. Use this blog post as a starting point, but make sure you customize the list for the given project.
+
+## My current checklist
+Here's my current checklist. I didn't list all the stories as above (this would've made way to long of a checklist), but I hope it's helpful for others!
+
+### Power/Battery
+
+- [ ] How does the firmware power off the device? Especially for a battery powered device.[1](#story-1-brown-out-loop)
```

Comment:
To clarify, is this asking "how does the firmware shut off power to the device, particularly if it is battery-powered?" or "how does the MCU get power from the power source, particularly if it is a battery?" I'm guessing the first but want to be sure because "power off" could have a few interpretations

Reply:
Switched wording to "shutdown".